docker-compose.yml

```diff
@@ -24,23 +26,17 @@ services:
       - REFORM_SERVICE_NAME
       - REFORM_TEAM
       - REFORM_ENVIRONMENT
-    volumes:
-      - ./build/install/job-scheduler:/opt/app/
     ports:
-      - 8484:8484
+      - $SERVER_PORT:$SERVER_PORT
```

Comment:
As port is application-specific, I think it would be best to leave a fixed value. Otherwise you would have to change the environment variable every time you start working on a different app.

Reply:
it is fixed value. in the `.env` which is also used for swagger docs publish. is not overridden by export either. test trace:

```bash
export SERVER_PORT=9999

./bin/run-in-docker.sh
....
Tomcat started on port(s): 8484 (http)
```

and there is no reference of 9999 anywhere

Reply:
I missed the fact that it's configured this way 👍